src/ADAL.PCL.WinRT/PlatformInformation.cs

```diff
@@ -81,6 +81,11 @@ public override string GetOperatingSystem()
 
         public override string GetDeviceModel()
         {
+            return null;
+        }
+
+        public override string GetDeviceId()
```

Comment:
SKU stands for Stock Keeping Unit for the deviceid. https://msdn.microsoft.com/en-us/library/windows/apps/windows.security.exchangeactivesyncprovisioning.easclientdeviceinformation.systemsku